lib/compat/wordpress-6.8/blocks.php

```diff
@@ -170,7 +170,7 @@ function gutenberg_apply_block_hooks_to_post_content( $content ) {
  * @return WP_REST_Response The response object.
  */
 function gutenberg_insert_hooked_blocks_into_rest_response( $response, $post ) {
-	if ( empty( $response->data['content']['raw'] ) || empty( $response->data['content']['rendered'] ) ) {
```

Comment:
Turns out that synced patterns don't have rendered content set, so I had to loosen the criterion here.